src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -709,6 +679,19 @@ void CC32xxWiFi::run_on_network_thread(std::function<void()> callback)
 }
 
 /*
+ * CC32xxWiFi::fd_remove()
+ */
+void CC32xxWiFi::fd_remove(int16_t socket)
```

Comment:
del_sockets calls the new_highest() function but the new_highest() function will never reduce fdHighest. This is a bug / performance problem -- the code does not crash as is but then the new_highest() call is unnecessary.